core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/RepairStateImpl.java

```diff
@@ -170,17 +170,15 @@ private long calculateRepairedAt(final VnodeRepairStates vnodeRepairStates, fina
 
     private long repairedTableRepairedAt(final long repairedAt, final RepairStateSnapshot old)
     {
-        if (LOG.isInfoEnabled())
+        if (LOG.isDebugEnabled())
         {
             long next = repairedAt + myRepairConfiguration.getRepairIntervalInMs();
             if (old != null)
             {
                 next -= old.getEstimatedRepairTime();
             }
-            LOG.info("Table {} last repaired at {}, next repair {}",
-                    myTableReference,
-                    MY_DATE_FORMAT.get().format(new Date(repairedAt)),
-                    MY_DATE_FORMAT.get().format(new Date(next)));
+            LOG.debug("Table {} fully repaired at {}, next repair at/after {}", myTableReference,
```

Comment:
Not sure how to leave comments on unchanged lines, but should the newTableRepairedAt() method also be aligned to DEBUG level? And addition of the word "at"? Looking at the calculations on lines 206-208 + 214, I don't think the "next repair" can ever be in the past, so "after" would not be applicable. Though, like I said above, the meaning of the entry doesn't change that much to me, regardless of using "at" or "at/after".

Reply:
I left it at info because this only happens if you don't have repair history, i.e it will only happen once. It can be good for user to know when the repair is scheduled to run.